src/main/webapp/app/shared/user-settings/user-settings-container/user-settings-container.component.html

```diff
@@ -31,13 +31,15 @@
                 <a class="list-group-item btn btn-outline-primary" routerLink="science" routerLinkActive="active" jhiTranslate="artemisApp.userSettings.scienceSettings"></a>
                 @if (localVCEnabled) {
                     <a class="list-group-item btn btn-outline-primary" routerLink="ssh" routerLinkActive="active" jhiTranslate="artemisApp.userSettings.sshSettings"></a>
-                    <a
-                        class="list-group-item btn btn-outline-primary"
-                        routerLink="vcs-token"
-                        routerLinkActive="active"
-                        jhiTranslate="artemisApp.userSettings.vcsAccessTokenSettings"
-                    >
-                    </a>
+                    @if (isAtLeastTutor) {
```

Comment:
TODO maybe also for manual reviewers: This only hides the link, if I understand correctly. Would the actual settings page still be accessible via a direct link to `$ARTEMIS/user-settings/vcs-token`?

Reply:
Yep it would still be accessible. But that's not really an issue here. The only reason we want students to not be able to see this, is to not confuse them, especially since they do not need such tokens atm.